bundles/org.openhab.ui/web/src/components/thing/channel-link.vue

```diff
@@ -61,6 +62,8 @@
 import AddLinkPage from '@/pages/settings/things/link/link-add.vue'
 import ConfigureLinkPage from '@/pages/settings/things/link/link-edit.vue'
 import ConfigureChannelPage from '@/pages/settings/things/channel/channel-edit.vue'
+import AddChannelPage from '@/pages/settings/things/channel/channel-add.vue'
```

Comment:
Seems like this is not used at all. If this is correct, can you please remove this import?

Reply:
ok, done